site/content/contribute/server/feature-flags.md

```diff
@@ -26,13 +26,115 @@ Feature flags allow us to be more confident in shipping features continuously to
-2. Once created, set the treatment values appropriately. The defaults of "on" an "off" can work for most feature flags.
-3. When ready to deploy to cloud, set the targeting rules appropriately to slowly roll out as required. 
+We've used their SDK to integrate with the Mattermost server. You can find the [Split documentation here](https://help.split.io/hc/en-us).
+
+### Who has access to Split?
+
+We currently have a limited number of seats for our Split account so not everyone gets access. Currently, 2 people per engineering team plus some PM/UX folk will have access. If you need to manage some feature flags, ask your team lead who has access and can help you out. It's up to each team to determine who has access.
+
+### How to log in?
+
+Login is controlled via OneLogin. If you have access you will be able to log in through there.
+### Terminology
+
+In Split, there's some terminology you will need to understand:
+
+ - **Split** - A feature flag
+ - **Treatment** - The values a split (e.g. feature flag) can have
+ - **Environment** - Different environments that splits can target such as test, staging and production
+ - **Server** - Single Mattermost workspace
+ - **Segment** - A list of servers that can be targeted by a split
+
+### Adding a split
+
+After you've added the feature flag in code, you'll need to create a split for it.
+
+ 1. Log in to Split.
+ 2. On left sidebar, click "Splits".
+ 3. At the top click "Create split".
+ 4. Fill in the following:
+   1. Enter the name of your feature flag. It must match the name used in code in `model/feature_flags.go`.
+   2. Select "server" as the traffic type.
+   3. Optionally add tags and a description.
+   4. Click "Create".
+
+The split is now created but not targeting anything. The next section covers adding basic targeting rules.
+
+###  Add targeting rules to a split
+
+To have the split have any effect, it needs targeting rules to know which environment and servers it should set the feature flag value to. To add basic targeting rules:
+
+ 1. Find your split and click on it.
+ 2. At the top, under the name of the split select the "Environment" you want to target.
+   - See [Environments](#Environments) for a description of each.
+ 3. In the center of the screen, select "Add Rules".
+ 4. Define your treatments:
+   - Recall that treatments are just the values your feature flag can have.
+   - Boolean flags should use the default "on" and "off" treatments.
+   - For non-boolean flags, change the treatments to be the possible values. E.g. for a split that specifies a plugin's version you could have three treatments of "1.0", "1.1" and "2.0".
+ 5. Down the page a bit, set the default rule to one of your treatments.
+   - This will set the flag for all servers in the environment while you don't have any other targeting rules configured.
+ 6. At the top right, click "Save changes" and confirm them on the next screen.
```

Comment:
```suggestion
     - See [Environments](#Environments) for a description of each.
 3. In the center of the screen, select **Add Rules**.
 4. Define your treatments:
     - Recall that treatments are just the values your feature flag can have.
     - Boolean flags should use the default "on" and "off" treatments.
     - For non-boolean flags, change the treatments to be the possible values. E.g. for a split that specifies a plugin's version you could have three treatments of "1.0", "1.1" and "2.0".
 5. Down the page a bit, set the default rule to one of your treatments.
     - This will set the flag for all servers in the environment while you don't have any other targeting rules configured.
 6. At the top right, click **Save changes** and confirm them on the next screen.
```